src/Clock.tsx

```diff
@@ -0,0 +1,59 @@
+import { Component } from 'react';
+
+type Props = {
+  clockName: string
+};
+
+type State = {
+  today: string
+};
+
+export class Clock extends Component<Props> {
+  state: Readonly<State> = {
+    today: new Date().toUTCString().slice(-12, -4),
```

Comment:
It's better to create a helper function outside the Clock component, f.e. `getFormattedTime` and reuse it